fcp-0010.md

```diff
@@ -0,0 +1,44 @@
+---
+authors: Warner Losh <imp@bsdimp.com>
+state: draft
+---
+
+# FCP 10: PC Card Deprecation Notice
+
+PC Card has not been relevant in years. It's inclusion in the tree is holding us back.
+
+## Problem Statement
+
+PC Card used to be very relevant. However, the only reason we have a
+number of really old network drivers is due to PC Card. If we were to
+remove it, many of them could be deleted. PC Card is a 16-bit only
+standard. Most relevant hardware that FreeBSD runs on has CardBus
```

Comment:
Maybe "Most relevant hardware FreeBSD runs on that have PC Card slots have CardBus slots"?  As it is, the sentence implies that most hardware FreeBSD runs on has CardBus slots and that's definitely not true and not something I think you mean to say.

Reply:
I'll clarify. Relevant here referred to hardware that had pcmcia slots, not relevant to the project.